app_core.py

```diff
@@ -120,6 +120,11 @@ def default(self, o):
 else:
     app.config["CSS_THEME_INTENSITY"] = "light"
 
+if os.getenv("MIN_AVAILABLE_NZD_BALANCE_CENTS"):
```

Comment:
might as well rename this to `MIN_AVAILABLE_NZD_BALANCE` and parse it as a decimal

Reply:
changed the name and the value to decimal.